deploy/deploy-basic.ts

```diff
@@ -1,20 +1,75 @@
 import "@nomiclabs/hardhat-ethers"
+const color = require("cli-color")
+var msg = color.xterm(39).bgXterm(128)
+import hre, { ethers, network } from "hardhat"
 
-//your address here...
 const initialMint = ethers.parseEther("10000")
 
-//to deploy, run yarn hardhat deploy --network hardhat
-
-export default async ({ getNamedAccounts, deployments }) => {
+export default async ({ getNamedAccounts, deployments }: any) => {
     const { deploy } = deployments
 
     const { deployer } = await getNamedAccounts()
     console.log(deployer)
 
-    await deploy("Basic", {
+    const basic = await deploy("Basic", {
         from: deployer,
         args: [initialMint],
         log: true
     })
+
+    console.log("basic.receipt.contractAddress:", basic.receipt.contractAddress)
+    console.log("hre.network.name:", hre.network.name)
+
+    switch (hre.network.name) {
+        case "arthera-testnet":
+            console.log(
+                "Basic ERC-20 token contract deployed:",
+                msg(basic.receipt.contractAddress)
+            )
+
+            try {
+                // Please use `pnpm sourcify:arthera` after the deployment instead.
+
+                // console.log("\nEtherscan verification in progress...")
+                // console.log(
+                //     "\nWaiting for 6 block confirmations (you can skip this part)"
+                // )
+                // await basic.deploymentTransaction()?.wait(6)
+                // await hre.run("verify:verify", {
+                //     network: network.name,
+                //     address: basic.receipt.contractAddress,
+                //     constructorArguments: [initialMint]
+                // })
+
+                console.log(
+                    "Please use `pnpm sourcify:arthera` to verify your contract."
+                )
+            } catch (error) {
+                console.error(error)
+            }
+
+            break
+        case "sepolia":
+            try {
+                console.log(
+                    "Basic ERC-20 token contract deployed:",
+                    msg(basic.receipt.contractAddress)
+                )
+                console.log("\nEtherscan verification in progress...")
+                console.log(
```

Comment:
you have commented the wait for 6 blocks confirmation so you should also comment the console log about it.